src/.nuget/packages.Windows_NT.config

```diff
@@ -1,5 +1,7 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00087" />
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta7" />
+  <!-- This version should be kept up to date with dir.props -->
+  <package id="NuProj" version="0.10.4-beta-gf7fc34e7d8" />
```

Comment:
Why here as opposed to in the project.json for the task library? Given that the task library is the one collecting and publishing these I would put it there.